wiki/Help_Centre/en.md

```diff
@@ -303,7 +302,7 @@ See [this topic](https://answers.microsoft.com/en-us/windows/forum/windows_vista
 
 #### Some of my beatmaps are missing!
 
-**Make sure you are not grouping your beatmaps by anything in the top right of the screen, "grouping" should be set to "no grouping".**
+**Make sure you are not grouping your beatmaps by anything in the top right of the screen, i.e. "grouping" should be set to "no grouping".**
```

Comment:
```suggestion
**Make sure you are not grouping your beatmaps by anything in the top right of the screen (i.e. "grouping" should be set to "no grouping".)**
```
i.e is grammatically fine after a comma but it's just a bit odd

Reply:
Alright, but the period should be outside the parentheses.

Reply:
True, thanks again:)